### PR TITLE
Facilities

### PR DESCRIPTION
**Pull request by shayansiddiqui**
## Issue

This fixes the following issue(s):

### CODE REVIEW

**Comment by kordianbruck** [comment]
@shayansiddiqui please describe what this PR is about. Set a better title and maybe add a screenshot of the added layout if you can. Thanks!

**Comment by kordianbruck** [comment]
@shayansiddiqui whats your progress?

File: app/src/main/java/de/tum/in/tumcampusapp/tumonline/TUMRoomFinderRequest.java [hunk 1]
```diff
@@ -189,6 +242,44 @@ public void cancelRequest(boolean mayInterruptIfRunning) {
         return roomsList;
     }
 
+
+    public List<Map<String, String>> fetchFacilities(String searchString) {
+//        TODO:replace with actual search call
+        String url = API_URL_SEARCH_FACILITIES + encodeUrl(searchString);
+//        Optional<JSONArray> jsonArray = net.downloadJsonArray(url, CacheManager.VALIDITY_DO_NOT_CACHE, true);
+        Optional<JSONArray> jsonArray=null;
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/tumonline/TUMRoomFinderRequest.java]: Please don't ever set an Optional = null. That's what Optional.absent() is for

**Comment by kordianbruck** [comment]
@shayansiddiqui please add screenshots here and commit your code!

**Comment by kordianbruck** [comment]
Right now I don't see how we can merge this. Closing this for now.

**Pull request closed without merging**